.github/workflows/lint.yml

```diff
@@ -7,12 +7,7 @@
 
 name: Lint
 
-on:
-  push:
-    branches: [ master, staging, dev ]
-  pull_request:
```

Comment:
Is there a reason for removing the `lint` pass from PRs?

Reply:
@thypon I've made it so that it runs on every push to the remote. I had another PR open that was making these changes anyway so I just merged them together.  From a workflow perspective it is actually much more useful to have the basic linting tasks run a lot.

Reply:
It seems the checks are somehow shown in this PR as well, so it does not seem to affect the output.

Reply:
@thypon The linting checks were run when I pushed the branch to the remote repository, the other checks were triggerred by me opening the PR.